### PR TITLE
Add docs meta to custom rules

### PR DESCRIPTION
**Pull request by kaelig**
cc @lemonmade @ismail-syed

### CODE REVIEW

File: lib/rules/binary-assignment-parens.js [hunk 1]
```diff
@@ -8,7 +8,14 @@ const BOOLEAN_OPERATORS = ['==', '===', '!=', '!==', '>', '>=', '<', '<='];
 
 module.exports = {
   meta: {
-    docs: {},
+    docs: {
+      description:
+        'Enforce (or disallow) assignments of binary, boolean-producing expressions to be wrapped in parentheses.',
+      category: 'Possible Errors',
```
Comment (lemonmade) [inline lib/rules/binary-assignment-parens.js]: Best practices I think

File: lib/rules/class-property-semi.js [hunk 2]
```diff
@@ -1,6 +1,12 @@
 module.exports = {
   meta: {
-    docs: {},
+    docs: {
+      description: 'Require (or disallow) semicolons for class properties.',
+      category: 'Possible Errors',
```
Comment (lemonmade) [inline lib/rules/class-property-semi.js]: Stylistic Issues

File: lib/rules/no-fully-static-classes.js [hunk 3]
```diff
@@ -1,6 +1,13 @@
 module.exports = {
   meta: {
-    docs: {},
+    docs: {
+      description:
+        'Prevent the declaration of classes consisting only of static members.',
+      category: 'Possible Errors',
```
Comment (lemonmade) [inline lib/rules/no-fully-static-classes.js]: Best Practices

File: lib/rules/no-useless-computed-properties.js [hunk 4]
```diff
@@ -1,6 +1,12 @@
 module.exports = {
   meta: {
-    docs: {},
+    docs: {
+      description: 'Prevent the usage of unnecessary computed properties.',
+      category: 'Possible Errors',
```
Comment (lemonmade) [inline lib/rules/no-useless-computed-properties.js]: Probably Best Practices too

File: lib/rules/prefer-class-properties.js [hunk 5]
```diff
@@ -1,6 +1,13 @@
 module.exports = {
   meta: {
-    docs: {},
+    docs: {
+      description:
+        'Prefer class properties to assignment of literals in constructors.',
+      category: 'Possible Errors',
```
Comment (lemonmade) [inline lib/rules/prefer-class-properties.js]: Stylelistic Issues

File: lib/rules/prefer-early-return.js [hunk 6]
```diff
@@ -2,7 +2,14 @@ const defaultMaximumStatements = 1;
 
 module.exports = {
   meta: {
-    docs: {},
+    docs: {
+      description:
+        'Prefer early returns over full-body conditional wrapping in function declarations.',
+      category: 'Possible Errors',
```
Comment (lemonmade) [inline lib/rules/prefer-early-return.js]: Best Practices

File: lib/rules/prefer-twine.js [hunk 7]
```diff
@@ -1,6 +1,12 @@
 module.exports = {
   meta: {
-    docs: {},
+    docs: {
+      description: 'Prefer Twine over Bindings as the name for twine imports.',
+      category: 'Possible Errors',
```
Comment (lemonmade) [inline lib/rules/prefer-twine.js]: Best Practices

File: lib/rules/restrict-full-import.js [hunk 8]
```diff
@@ -1,6 +1,12 @@
 module.exports = {
   meta: {
-    docs: {},
+    docs: {
+      description: 'Prevent importing the entirety of a package.',
+      category: 'Possible Errors',
```
Comment (lemonmade) [inline lib/rules/restrict-full-import.js]: Best Practices

File: lib/rules/sinon-no-restricted-features.js [hunk 9]
```diff
@@ -17,7 +17,13 @@ function createPropertyMatcher(properties) {
 
 module.exports = {
   meta: {
-    docs: {},
+    docs: {
+      description: 'Restrict the use of specified sinon features.',
+      category: 'Possible Errors',
```
Comment (lemonmade) [inline lib/rules/sinon-no-restricted-features.js]: Best Practices

File: lib/rules/sinon-prefer-meaningful-assertions.js [hunk 10]
```diff
@@ -54,7 +54,14 @@ const PREFER_SINON_ASSERT_MESSAGE =
 
 module.exports = {
   meta: {
-    docs: {},
+    docs: {
+      description:
+        'Require the use of meaningful sinon assertions through sinon.assert or sinon-chai.',
+      category: 'Possible Errors',
```
Comment (lemonmade) [inline lib/rules/sinon-prefer-meaningful-assertions.js]: Best Practices

**Review by lemonmade — APPROVED** [review]
A bunch of category changes, otherwise LGTM 👍

**Comment by kaelig** [comment]
Thanks, that's exactly the type of feedback I wanted!

**Pull request merged**